docs/content/comparisons/druid-vs-sql-on-hadoop.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: doc_page
+---
+
+Druid vs SQL-on-Hadoop (Hive/Impala/Drill/Spark SQL/Presto)
+===========================================================
+
+Druid is much more complementary to SQL-on-Hadoop engines than it is competitive. SQL-on-Hadoop engines provide an 
+execution engine for various data formats and data stores, and 
+many can be made to push down computations down to Druid, while providing a SQL interface to Druid.
+
+For a direct comparison between the technologies and when to only use one or the other, the basically comes down to your 
+product requirements and what the systems were designed to do.  
+
+Druid was designed to
+
+1. be an always on service
+1. ingest data in real-time
+1. handle slice-n-dice style ad-hoc queries
+
+SQL-on-Hadoop engines (as far as we are aware) were designed to replace Hadoop MapReduce with another, faster, query layer 
+that is completely generic and plays well with the other ecosystem of Hadoop technologies.
+
+What does this mean?  We can talk about it in terms of three general areas
+
+1. Queries
+1. Data Ingestion
+1. Query Flexibility
+
+## Queries
```

Comment:
This section should mention Parquet as a popular file format for speeding up queries.